src/StabilizerProcessor.cs

```diff
@@ -15,72 +15,72 @@ namespace chp
 {
     public class StabilizerProcessor : QuantumProcessorBase
     {
-        private bool[,] Table;
-        private int NQubits;
+        private readonly bool[,] table;
+        private readonly int nQubits;
```

Comment:
This would mean we can't dynamically add a qubit to the array, but with making it 1024 we are probably ok for now :P

Reply:
Yes, for now, untill we make it autosizing.